nipy/algorithms/registration/groupwise_registration.py

```diff
@@ -897,9 +914,11 @@ def __init__(self, images, tr, slice_times, slice_info,
         images : image or list of images
             Single or multiple input 4d images representing one or several fMRI
             runs.
-        tr : float
+        tr : None or float or string
```

Comment:
I'd prefer `None or float or 'header-allow-1.0'` to make the possible strings specific.